parser.js

```diff
@@ -981,11 +981,11 @@
         // ControlEscape
         var codePoint = 0;
         switch (res[0]) {
-          case 't': codePoint = 0x009; break;
-          case 'n': codePoint = 0x00A; break;
-          case 'v': codePoint = 0x00B; break;
-          case 'f': codePoint = 0x00C; break;
-          case 'r': codePoint = 0x00D; break;
+        case 't': codePoint = 0x009; break;
```

Comment:
This is odd, `case` should be more indented than `switch`.

Reply:
@liuxingbaoyu , can you do something about this one? I agree with @nicolo-ribaudo that the indention is odd - the `case` should be more indented than the `switch`.

Reply:
It has been fixed!